streams/src/test/java/org/apache/kafka/streams/integration/utils/EmbeddedKafkaCluster.java

```diff
@@ -123,11 +123,16 @@ private void putIfAbsent(final Properties props, final String propertyKey, final
      * Stop the Kafka cluster.
      */
     public void stop() {
-        for (final KafkaEmbedded broker : brokers) {
-            broker.stop();
+        try {
+            for (final KafkaEmbedded broker : brokers) {
+                broker.stop();
+            }
+        } catch (Exception e) {
+            log.warn(e.getMessage());
```

Comment:
This is catching any exception to stop brokers. I think the tests should ensure that broker was shutdown and any files/folders deleted. If ZK needs to be shutdown first to delete the folders, it may be better to split stop and delete into separate methods and still fail if either fails.